std/algorithm/iteration.d

```diff
@@ -889,6 +884,7 @@ template each(alias pred = "a")
     }
 }
 
+///
 unittest
```

Comment:
Ddoc'd unittest examples need to be vetted to make sure they make sense to the end user. Comments like "this is not unittestable", etc., don't make any sense from the user's POV and should not be included in ddoc'd unittests.  They should either be left as undocumented unittests, or edited so that it makes sense to someone reading the docs.